### PR TITLE
:sparkles: Trigger clusteradm e2e after post images.

### PR DESCRIPTION
**Pull request by xuezhaojun**
## Related PR
https://github.com/open-cluster-management-io/clusteradm/pull/352

### CODE REVIEW

File: .github/workflows/post.yml [hunk 1]
```diff
@@ -133,3 +132,13 @@ jobs:
...
+  trigger-clusteradm-e2e:
+    needs: [ images, image-manifest ]
+    name: trigger clusteradm e2e
+    runs-on: ubuntu-latest
+    steps:
+      - uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ secrets.OCM_BOT_PAT }}
+          repository: open-cluster-management-io/clusteradm
+          event-type: ocm_changes
```
Comment (zhujian7) [inline .github/workflows/post.yml:144]: can we also send the commit sha, so clusteradm will know which commit triggers?

Comment (xuezhaojun) [reply .github/workflows/post.yml:144]: Sure, the sha and ref added.

File: .github/workflows/post.yml [hunk 2]
```diff
@@ -133,3 +132,14 @@ jobs:
...
           docker manifest push quay.io/open-cluster-management/placement:latest
           docker manifest push quay.io/open-cluster-management/addon-manager:latest
+  trigger-clusteradm-e2e:
+    needs: [ images, image-manifest ]
+    name: trigger clusteradm e2e
+    runs-on: ubuntu-latest
+    steps:
+      - uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ secrets.OCM_BOT_PAT }}
```
Comment (xuezhaojun) [inline .github/workflows/post.yml:142]: Hi, @mikeshng does OCM_BOT_PAT has write permission on `Contents`? The permission is needed to send a repository-dispatch event.
image

**Pull request merged**